site/networking.xml

```diff
@@ -0,0 +1,698 @@
+      </doc:subsection>
+
+      <doc:subsection name="os-tuning-max-open-file-handles">
+      </doc:subsection>
+    </doc:section>
+
+    <doc:section name="socket-gen-tcp-options">
+      <doc:heading>TCP Socket Options</doc:heading>
+
+      <doc:subsection name="socket-gen-tcp-options-common">
+        <doc:heading>Common Options</doc:heading>
+        <dl>
+          <dt><code>rabbit.tcp_listen_options.nodelay</code></dt>
+          <dd>
+            When enabled, disables
+            <a href="http://en.wikipedia.org/wiki/Nagle's_algorithm">Nagle's algorithm</a>.
+            Default is true. Highly recommended for most users.
+          </dd>
+          <dt><code>rabbit.tcp_listen_options.sndbuf</code></dt>
+          <dd>
+            See TCP buffers discussion earlier in this guide. Default value is
+            automatically tuned by the OS, typically in the 88K to 128K range on
+            modern Linux versions. Increasing buffer size improves consumer throughput
+            and RAM use for every connection. Decreasing has the opposite effect.
+          </dd>
+          <dt><code>rabbit.tcp_listen_options.recbuf</code></dt>
+          <dd>
+            See TCP buffers discussion earlier in this guide. Default value effect
+            are similar to that of <code>rabbit.tcp_listen_options.sndbuf</code> but
+            for publishers and protocol operations in general.
+          </dd>
+          <dt><code>rabbit.tcp_listen_options.backlog</code></dt>
+          <dd>
+            Maximum size of the unaccepted TCP connections queue. When this size
+            is reached, new connections will be rejected. Set to 4096 or higher for
+            environments with thousands of concurrent connections and possible bulk client
+            reconnections.
+          </dd>
+          <dt><code>rabbit.tcp_listen_options.linger</code></dt>
+          <dd>
+            When set to <code>{true, N}</code>, sets
+            the timeout in seconds for flushing unsent data when the (server) socket is
+            closed.
+          </dd>
+          <dt><code>rabbit.tcp_listen_options.keepalive</code></dt>
+          <dd>
+            When set to <code>true</code>, enables TCP keepalives (see above).
+            Default is <code>false</code>. Makes sense for environments where
+            connections can go idle for a long time (at least 10 minutes),
+            although <a href="/heartbeats.html">heartbeats</a> is still recommended over
+            this option.
+          </dd>
+          <dt><code>rabbit.tcp_listen_options.reuseaddr</code></dt>
+          <dd>
+            Sets <code>SO_REUSEADDR</code> on the sockets, which allows a port belonging
+            to a connection in the <code>TIME_WAIT</code> state to be reused for other
+            connections. Enabled by default, highly recommended for environments with
+            large numbers of concurrent connections and/or connection churn (short-lived
+            clients).
```

Comment:
This option only affects the listening socket (ie. what should we do when RabbitMQ is restarted: if this is `false`, RabbitMQ will probably fail to bind the TCP port without waiting for several seconds), not the accepted connections.

I believe we do not need to document this variable in this guide.